firstpartydata/tests/extractfpdforbidders/global-fpd-defined-site.json

```diff
@@ -0,0 +1,74 @@
+{
+  "description": "req.site and req.ext.prebid.data.bidders contains appnexus; expect bidder config with site data to be included in the resolved FPD for appnexus",
```

Comment:
This test is more about site presence in resolved fpd because `req.ext.prebid.data.bidders` is defined and has config for bidder.  I renamed it in a little different way, please check.